lib/oauth2.js

```diff
@@ -49,25 +46,9 @@ exports.OAuth2.prototype.buildAuthHeader= function(token) {
   return this._authMethod + ' ' + token;
 };
 
-exports.OAuth2.prototype._chooseHttpLibrary= function( parsedUrl ) {
```

Comment:
not necessary anymore, since [request](https://github.com/request/request) handles that for us.